app/geo/gui/src/ts/twxplore/gui/geo/components/map/Map.tsx

```diff
@@ -271,7 +270,7 @@ const MapImpl: React.FunctionComponent = () => {
           //A query is not in progress AND if the length of the previous query was capped by the limit variable that was passed...
           if (
             featureLoadingState.latestQueryLength === limit &&
-            (featureLoadingState.offset < 2000 || !DEBUG)
+            (featureLoadingState.offset < 5000 || !DEBUG)
```

Comment:
Magic numbers should go in const's at file level so it's clear what they are. DEBUG_FEATURES_PER_TYPE_MAX = 5000 or something like this.